### PR TITLE
BEMHTML: Make XHTML mode optional. #201

### PR DESCRIPTION
**Pull request by tadatuta**

### CODE REVIEW

File: lib/bemhtml/index.js [hunk 1]
```diff
@@ -3,8 +3,9 @@ var utils = require('../bemxjst/utils');
 var Entity = require('./entity').Entity;
 var BEMXJST = require('../bemxjst');
 
-function BEMHTML() {
+function BEMHTML(options) {
   BEMXJST.apply(this, arguments);
+  this._shortTagCloser = options.xhtml === true ? '/>' : '>';
```
Comment (zxqfox) [inline lib/bemhtml/index.js]: кажется, что переменная (не свойство) будет дешевле

**Comment by miripiruni** [comment]
@tadatuta I would suggest get rid of `this._shortTagCloser` and merge.

**Comment by tadatuta** [comment]
I thought of `xhtml = false` by default in new major version (as there's no any reason in extra `/`) but you already released 6.0.0 :)

So what'd you suggest instead of `this._shortTagCloser`?

**Comment by miripiruni** [comment]
In order to make this code faster you can leave `this._shortTagCloser`.

Can we change default value of `xhtml` to `true` and release it as `5.2`?

And in 6.1 we can inverse `xhtml`.

**Comment by tadatuta** [comment]
> And in 6.1 we can inverse xhtml.

Not sure if it's ok to make such changes in minor releases (even though 6.0 was not a real major one). Let's postpone inversion till 7.0?

**Comment by miripiruni** [comment]
@tadatuta oh, yes, of course 7.0

**Comment by tadatuta** [comment]
:up:

**Comment by miripiruni** [comment]
:+1:

**Comment by miripiruni** [comment]
May I merge?

**Comment by tadatuta** [comment]
if you asking me — I'm fine with it :)

File: test/bemhtml-test.js [hunk 2]
```diff
@@ -104,4 +105,48 @@ describe('BEMHTML engine tests', function() {
       block: 'b1'
     }, '<div class="b1">ok</div>');
   });
+
+  describe('xhtml option', function() {
+    it('should close short tags in BEMJSON by default', function() {
+      compile('')
+        .apply({ tag: 'br' })
+        .should.equal('<br/>');
+    });
```
Comment (veged) [inline test/bemhtml-test.js]: should be enough to make just 2 `it`'s about default with BEMJSON and `false` with BEMHTML

**Pull request merged**